src/main/java/uk/gov/hmcts/reform/civil/model/genapplication/GeneralApplication.java

```diff
@@ -43,6 +43,7 @@ public class GeneralApplication implements MappableObject {
     private final String defendant1PartyName;
     private final String defendant2PartyName;
     private final String generalAppSuperClaimType;
+    private final GACaseManagementCategory caseManagementCategory;
```

Comment:
If we declare this as a DynamicList then we will not need additional GACaseManagementCategory

Reply:
yes, you are right. I just followed the same format what civil has followed for future proof.